diff --git a/index.d.ts b/index.d.ts
index 318791d52a1a3ea4789d8ee414863e6cc09a494b..6b3da2beedc704ef4c1bd1f47bad45d6af787894 100755
--- a/index.d.ts
+++ b/index.d.ts
@@ -148,7 +148,7 @@ declare namespace Database {
         ? BetterSqlite3.Statement<BindParameters>
         : BetterSqlite3.Statement<[BindParameters]>;
     type ColumnDefinition = BetterSqlite3.ColumnDefinition;
-    type Transaction = BetterSqlite3.Transaction<VariableArgFunction>;
+    type Transaction<T extends VariableArgFunction = VariableArgFunction> = BetterSqlite3.Transaction<T>;
     type Database = BetterSqlite3.Database;
 }
 
